jvm-libs/linea/linea-contracts/l1-rollup/build.gradle

```diff
@@ -16,8 +16,10 @@ dependencies {
 
 web3jContractWrappers {
   def contractAbi = layout.buildDirectory.dir("${rootProject.projectDir}/contracts/abi").get()
-    .file("LineaRollupV5.0.abi").asFile.absolutePath
+    .file("LineaRollupV6.0.abi").asFile.absolutePath
```

Comment:
will this break if V5 is selected, would this not also be dynamic?

Reply:
Not it won't break, for several 2 reasons:
- coordinator dynamically supports V5 and V6
- coordinator is fetching V5 from maven, see [here](https://github.com/Consensys/linea-monorepo/pull/240/files#diff-4f467aec2ca69e8e36de0261ce59ff6edf6cff724f18aaeb961ffaef5b9585f4R8-R10)
- Once V6 is final, we will release v6 to Maven as well and lock the version, like we have for v5.